src/support/global.c

```diff
@@ -127,6 +127,7 @@ __global_once(void)
     /* Run-time configuration. */
 #ifdef WT_STANDALONE_BUILD
     __wt_process.fast_truncate_2022 = true;
+    __wt_process.page_stats_2022 = true;
```

Comment:
I think it is better if it starts with "false" till some testing is done in our local environments.

Reply:
Good idea, done.